### PR TITLE
Build CUBE landing page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build CUBE landing page at `/cube`

## QA

- Visit the demo at: https://ubuntu-com-8925.demos.haus/cube
- Check the page against this [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd355ed883fc9481de84a69)

## Issue / Card

Fixes [#3498](https://github.com/canonical-web-and-design/web-squad/issues/3498)

## Screenshots

image

### CODE REVIEW

**Comment by anasereijo** [comment]
🥇 looks goood

File: static/sass/_pattern_strip.scss [hunk 1]
```diff
@@ -811,6 +811,11 @@
       clip-path: polygon(0% 0%, 100% 0%, 100% calc(100% - 4rem), 0% 100%);
       padding-bottom: 6rem;
     }
+    &--top-right {
+      clip-path: polygon(0% 0%, 100% calc(0% + 4rem), 100% 100%, 0% 100%);
```
Comment (ClementChaumel) [inline static/sass/_pattern_strip.scss:815]:
```suggestion
      clip-path: polygon(0% 0%, 100% 4rem, 100% 100%, 0% 100%);
```

isn't `calc(0% + 4rem)` just `4rem`?

File: templates/cube/index.html [hunk 2]
```diff
@@ -27,55 +27,135 @@ <h1>You&rsquo;re a professional.</br>Prove it with CUBE.</h1>
...
 <section class="p-strip">
-  <div class="u-fixed-width">
-    <h2>What are Microcertifications?</h2>
-    <p class="p-heading--four">Earn your qualification one topic at a time, at your own pace, in any order.</p>
-    <p>The CUBE curriculum is perfectly portoned to accomodate your busy schedule. It consists of 15 separate microcertifications, each covering a specific subject area, that can be taken in any order and at any time.</p>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>What are Microcertifications?</h2>
+      <p class="p-heading--four">Earn your qualification one topic at a time, at your own pace, in any order.</p>
+      <p class="u-sv3">The CUBE curriculum is perfectly portoned to accomodate your busy schedule. It consists of 15 separate microcertifications, each covering a specific subject area, that can be taken in any order and at any time.</p>
```
Comment (ClementChaumel) [inline templates/cube/index.html:34]: Is portoned a typo for portioned or just a word I don't know? Google didn't return anything

**Review by ClementChaumel — APPROVED** [review]
A few nitpicks but it looks good to me! :+1:

File: templates/cube/index.html (hunk 2 above)
Comment (bethcollins92) [inline templates/cube/index.html:34]:
```suggestion
      <p class="u-sv3">The CUBE curriculum is perfectly portioned to accomodate your busy schedule. It consists of 15 separate microcertifications, each covering a specific subject area, that can be taken in any order and at any time.</p>
```

File: static/sass/_pattern_strip.scss (hunk 1 above)
Comment (bethcollins92) [reply static/sass/_pattern_strip.scss:815]: @ClementChaumel very true, I was just keeping the calc consistent with the other classes

File: templates/cube/index.html (hunk 2 above)
Comment (bethcollins92) [reply templates/cube/index.html]: Great spot, thanks!

**Pull request merged**